app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render 'common/navigation', :title => 'Categories' %>
```

Comment:
Please follow the same layout for every category card on `categories` page as in the design. Moreover, please use the background color for `Add new category` button from the design page.